src/components/home/vacations-card.tsx

```diff
@@ -143,11 +149,59 @@ const VacationsCard = () => {
     fetchVacationsRequests();
   }, []);
 
+  /**
+   * Initialize logged in person's data.
+   */
+  const getPersonVacations = async () => {
+    if (persons.length) {
+      setLoading(true);
+      if (loggedInPerson || config.person.id) setLoading(false);
+      {
+        const fetchedPersons = await personsApi.listPersons({ active: true });
+        setPersons(fetchedPersons);
+      }
+    }
+  };
+
+  useMemo(() => {
+    if (!persons) {
+      getPersonVacations();
+    }
+  }, [persons]);
```

Comment:
On row 156 you check for persons.length and on row 167 you check for !persons. These checks are in conflict. I think you meant to check for !persons.length in the function. Please test and verify if this is the case.

Reply:
removed second check.